nipype/interfaces/cat12/preprocess.py

```diff
@@ -0,0 +1,355 @@
+import os
+from pathlib import Path
+
+from nipype.interfaces.base import InputMultiPath, TraitedSpec, traits, isdefined, File, Str
+from nipype.interfaces.spm import SPMCommand
+from nipype.interfaces.spm.base import SPMCommandInputSpec, ImageFileSPM, scans_for_fnames, scans_for_fname
+from nipype.utils.filemanip import split_filename, fname_presuffix
+
+
+class CAT12SegmentInputSpec(SPMCommandInputSpec):
+    in_files = InputMultiPath(ImageFileSPM(exists=True), field="data", desc="file to segment", mandatory=True,
+                              copyfile=False)
+
+    _help_tpm = 'Tissue Probability Map. Select the tissue probability image that includes 6 tissue probability ' \
+                'classes for (1) grey matter, (2) white matter, (3) cerebrospinal fluid, (4) bone, (5) non-brain  ' \
+                'soft tissue, and (6) the background.  CAT uses the TPM only for the initial SPM segmentation.'
+    tpm = InputMultiPath(ImageFileSPM(exists=True), field="tpm", desc=_help_tpm, mandatory=False,
+                         copyfile=False)
+
+    n_jobs = traits.Int(1, usedefault=True, mandatory=True, field="nproc", desc="Number of threads")
+    use_prior = Str(field="useprior", usedefault=True)
```

Comment:
Missing descriptions for:
* use_prior
* surface_measures
* neuromorphometrics
* lpba40
* cobra
* hammers
* save_bias_corrected

Unclear descriptions:
* own_atlas
* gm_output_*
* wm_output_*
* csf_output_*
* label_* / output_labelnative
* las_*